### PR TITLE
feat: make categories a thing

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

File: content-organization/creating-tags/index.qmd [hunk 1]
```diff
@@ -4,37 +4,47 @@ title: "Creating Tags"
...
+client = connect.Client()
 response = client.post("/v1/tags", json={
     "name": "Departments",
 })
+
+departments = response.json()
 ```
 
-The response contains information on the created tag.
+The `departments` object additional information about the tag catgegory, including the `id` field. We will use the `id` field in the next step to create a tag.
```
Comment (kgartland-rstudio) [inline content-organization/creating-tags/index.qmd:43]: I think it's missing the word "contains" and category is misspelled.

"The `departments` object contains additional information about the tag category,"

Comment (tdstein) [reply content-organization/creating-tags/index.qmd:43]: good eye 
```suggestion
The `departments` object contains additional information about the tag category, including the `id` field. We will use the `id` field to create a tag in the next step.
```

File: content-organization/creating-tags/index.qmd [hunk 2]
```diff
@@ -49,51 +59,57 @@ shape: (1, 5)
...
-> depts_tag
+> departments
 Posit Connect Tag Tree (filtered)
 └── Departments
 ```
 
 :::
 
-## Discussion
+Next, create a tag and assign it to the tag category.
```
Comment (toph-allen) [inline content-organization/creating-tags/index.qmd:86]: Perhaps also a note that the method in this section can also be used to nest tags inside other tags, in addition to giving them a parent category.

File: content-organization/creating-tags/index.qmd [hunk 3]
```diff
@@ -49,51 +59,57 @@ shape: (1, 5)
 
 ```{.r}
 library(connectapi)
+
+TAG_CATEGORY_NAME = "Departments"
+
 client <- connect()
+create_tag(client, "TAG_CATEGORY_NAME")
```
Comment (kgartland-rstudio) [inline content-organization/creating-tags/index.qmd:66]: nit: since we're now creating tag categories, should this be `create_tag_category`?

**Review by kgartland-rstudio — APPROVED** [review]
Looks good!

File: content-organization/creating-tags/index.qmd (hunk 3 above)
Comment (tdstein) [reply content-organization/creating-tags/index.qmd:66]: I don't see a function for that in connectapi: https://github.com/search?q=repo%3Arstudio%2Fconnectapi%20create_tag_category&type=code

**Review by toph-allen — APPROVED** [review]
Looks good!

I feel like our documentation should explain tags better somewhere lol, I definitely misunderstood it when I edited this one. (i.e. Tags can be nested arbitrarily deep, it's just that the top level that can't be assigned to content which makes them categories.)

**Comment by tdstein** [comment]
> Looks good!
> 
> I feel like our documentation should explain tags better somewhere lol, I definitely misunderstood it when I edited this one. (i.e. Tags can be nested arbitrarily deep, it's just that the top level that can't be assigned to content which makes them categories.)

Agreed. I also think the common confusion is a sign that something else is wrong with the design. I've always wanted to call them folders/directories since that is what they act like.

**Comment by annikakrilov** [comment]
I have a few editing comments I would have liked to make here - when tagging reviewers, can we leave a little more time for everyone to get a review in before merging, or if urgent just shoot me a message on slack as things come through? I don't have popup email notifications/just check email periodically through the day so that I don't get disrupted for non-urgent things.

**Comment by tdstein** [comment]
> I have a few editing comments I would have liked to make here - when tagging reviewers, can we leave a little more time for everyone to get a review in before merging, or if urgent just shoot me a message on slack as things come through? I don't have popup email notifications/just check email periodically through the day so that I don't get disrupted for non-urgent things.

Yes, sorry about that @annikakrilov - I assumed you could still make review comments after a pr is closed, but that isn't the case. For these edits, are you able to make the edits as a separate pull request?

**Pull request merged**